Source/src/Program.h

```diff
@@ -12,6 +12,7 @@ namespace Hachiko
 
         void BindUniformFloat4x4(const char* name, const float* data, bool transpose = true) const;
```

Comment:
In the future we can have same method overriden for different types of data to have more clean looking codebase.

Reply:
I created just the thing for that
[Improve binding](https://github.com/AkitaInteractive/Hachiko-Engine/issues/36)

Reply:
Perfect, looking forward to it